Vagrantfile

```diff
@@ -1,11 +1,13 @@
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/focal64"
-  config.vm.network "private_network", ip: "192.168.56.100"
-  config.vm.network "forwarded_port", guest: 80, host: 80
-  config.vm.provider "virtualbox" do |v|
-    v.memory = 10240
-    v.cpus = 4
-  end
+config.vm.box = "ubuntu/focal64"
+config.vm.network "private_network", ip: "192.168.56.100"
+config.vm.network "forwarded_port", guest: 80, host: 80
+config.vm.provider "virtualbox" do |v|
+  v.memory = 10240
+  v.cpus = 4
+end
```

Comment:
Is this indentation change needed?

Reply:
Sorry I don't understand. Why does this change appears in the diff? If you did not changed it on purpose, please revert.